src/inimpkg/embedded.nim

```diff
@@ -1,4 +1,6 @@
 ## INim/src/embedded.nim: preloaded into user's REPL session
 ## ---------------------------------------------------------
-{. warning[UnusedImport]:off .}
+{.warning[UnusedImport]: off.}
```

Comment:
Can we reduce the scope of this somehow so we can catch genuine unused imports?

Reply:
Hmmm, it might be possible. It might be better to wait until we get external editor support, otherwise we'll see these unused import messages every time we run a line (it'll trigger every time we recompile the buffer file). When users can edit the file, they can remove any unused imports, but it's kinda difficult at the moment